include/deal.II/base/vectorization.h

```diff
@@ -3957,9 +3957,307 @@ operator<<(std::ostream &out, const VectorizedArray<Number, width> &p)
   return out;
 }
 
+//@}
+
+/**
+ * @name Ternary operations on VectorizedArray
+ */
+//@{
+
+
+/**
+ * enum class encoding binary operations for a component-wise comparison of
+ * VectorizedArray data types.
+ *
+ * @note In case of SIMD vecorization (sse, avx, av512) we select the
+ * corresponding ordered, non-signalling (<code>OQ</code>) variants.
+ */
+enum class SIMDComparison : int
+{
+#if DEAL_II_COMPILER_VECTORIZATION_LEVEL > 0
```

Comment:
This needs to be `#if DEAL_II_COMPILER_VECTORIZATION_LEVEL > 1 && defined(__AVX__)` because the relevant fields `_CMP_XX_OQ` are in `avxintrin.h`.

Reply:
fixed